projects/cmake/build.sh

```diff
@@ -108,6 +110,10 @@ if [ -n "$boost_lib" ] ; then
     cmake_args="-DBOOST_LIBRARYDIR=\"${boost_lib}\" $cmake_args"
 fi
 
+if [ -n "$boost_include" ] ; then
+    cmake_args="-DBOOST_INCLUDEDIR=${boost_include} $cmake_args"
+fi
+
 if [ "$static_boost" = "true" ] ; then
```

Comment:
In addition to describing the path to the include files, the user might want to specify 
`-DBoost_DEBUG=ON` and `-DBoost_VERBOSE=ON`.